test/lib/rails-jquery-autocomplete_test.rb

```diff
@@ -15,22 +15,22 @@ class ::Movie ; end
       end
 
       should 'respond to the action' do
-        assert_respond_to @controller, :autocomplete_movie_name
+        assert_respond_to @controller, :autocomplete_name
       end
 
       should 'render the JSON items' do
-        mock(@controller).get_autocomplete_items({
-          :model => Movie, :method => :name, :options => @options, :term => "query"
-        }) { @items }
+        mock(@controller).get_autocomplete_items(
+            { :model => Movie, :method => :name, :options => @options, :term => "query" }
```

Comment:
Line is too long. [89/80]<br>Redundant curly braces around a hash parameter.<br>Indent the first parameter one step more than the start of the previous line.<br>Use the new Ruby 1.9 hash syntax.